src/ir/local-graph.h

```diff
@@ -35,10 +36,12 @@ struct LocalGraph {
   // the constructor computes getSetses, the sets affecting each get
   LocalGraph(Function* func);
 
-  // the local.sets relevant for an index or a get.
-  typedef std::set<LocalSet*> Sets;
+  // The local.sets relevant for an index or a get. The most common case is to
+  // have a single set; after that, to be a phi of 2 items, so we use a small
+  // set of size 2 to avoid allocations there.
+  using Sets = SmallSet<LocalSet*, 2>;
 
-  typedef std::map<LocalGet*, Sets> GetSetses;
+  using GetSetses = std::unordered_map<LocalGet*, Sets>;
```

Comment:
How about a drive-by fix of `typedef`->`using` on line 46? (That's the only line left in this file with `typedef`)